grab.py

```diff
@@ -44,18 +74,12 @@ def add_timestamp(img):
         print('Usage: %s <webcamname>' % sys.argv[0])
         sys.exit(1)
 
-    # Get a pygame surface
-    frame = capture_frame()
-    if frame is None:
-        print('Could not capture frame. Is a webcam connected?')
-        sys.exit(1)
-
-    # Convert to a PIL image
-    img = cv_to_pil(frame)
-
-    # Add timestamp
-    add_timestamp(img)
+    # Remove existing images (in case camera was removed we don't want
+    # its old image hanging around).
+    for i in range(10):
```

Comment:
If I'm not mistaken, the number of images is dependent on the number of webcams attached. We can safely assume that there will not be 11 or more webcams attached to our Raspberry Pi.